portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -63,6 +118,28 @@ public Comment(Comment comment, long id, long timePosted) {
     this.timePosted = timePosted;
   }
 
+  /**
+   * Creates a comment object from a database entry.
+   * @param commentEntity
+   */
+  public Comment(Entity commentEntity) {
+    this.id = commentEntity.getKey().getId();
+    this.author = (String) commentEntity.getProperty(AUTHOR_KEY);
+    this.commentBody = (String) commentEntity.getProperty(BODY_KEY);
+    this.timePosted = (long) commentEntity.getProperty(TIME_POSTED_KEY);
+  }
+
+  /**
+   * Fills a datastore entity with the comment's data.
+   * @param commentEntity
+   */
+  public void fillEntity(Entity commentEntity) {
+    long timePosted = System.currentTimeMillis();
```

Comment:
If we make the change above for `timePosted`'s default value, do we still need this line?